pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -77,7 +78,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler, namespace string) error {
 	}
 
 	go func() {
-		for range ticker.C {
+		for ; true; <-ticker.C {
```

Comment:
@pb82 wdyt? The sleep just ensures that grafana has enough time to become available (for the initial request) the 10 second delay shouldn't have much of an impact on subsequent resyncs.